src/cls/IPM/Utils/Module.cls

```diff
@@ -1343,6 +1343,10 @@ ClassMethod LoadNewModule(
             tcommit
         }
         $$$ThrowOnError(tSC)
+        // On each load of a new module, we want to seed its update steps
+        if 'tDeveloperMode {
```

Comment:
Again, not here but rather in the activate phase in %IPM.Lifecycle.Base. Also, in GetCompletePhasesForOne(), update should run all phases that activate would, in addition to update at the end.